quest/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set version = "2.4.1" %}
+
+package:
+  name: quest
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/erdc/quest.git
+  git_rev: {{ version }}
```

Comment:
please build from a tarball https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos

Reply:
Why does it have to be built from a tarball? Your documents say that you can use git for a build here: http://192.168.1.1/ui/1.0.99.185832/dynamic/login.html.